tests/Rules.Framework.Tests/Evaluation/DataTypeConfigurationTests.cs

```diff
@@ -0,0 +1,49 @@
+namespace Rules.Framework.Tests.Evaluation
+{
+    using FluentAssertions;
+    using Rules.Framework.Core;
+    using Rules.Framework.Evaluation;
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+    using Xunit;
+
+    public class DataTypeConfigurationTests
+    {
+        [Fact]
+        public void Create_GivenDataTypeWithNonNullSystemTypeAndADefault_ReturnsNewInstance()
+        {
+            // Arrange
+            DataTypes dataType = DataTypes.Integer;
```

Comment:
use vars

Reply:
Same as explained before.

Reply:
Didn't understand. Can't we change the explicit variable declarations to var?

Reply:
My bad here, I've fixed as you requested.